rest/replicator_test.go

```diff
@@ -339,10 +339,8 @@ func TestActiveReplicatorPullAttachments(t *testing.T) {
 	assert.Equal(t, revID, doc2.SyncData.CurrentRev)
 	assert.Equal(t, "rt2", doc.GetDeepMutableBody()["source"])
 
-	// Because we're targeting a Hydrogen node that supports proveAttachment, we only end up sending the attachment once.
-	// If targeting a pre-hydrogen node, GetAttachment would be 2.
-	assert.Equal(t, int64(1), ar.Pull.GetStats().GetAttachment.Value())
-	assert.Equal(t, int64(1), ar.Pull.GetStats().ProveAttachment.Value())
+	assert.Equal(t, int64(2), ar.Pull.GetStats().GetAttachment.Value())
+	assert.Equal(t, int64(0), ar.Pull.GetStats().ProveAttachment.Value())
```

Comment:
Can we leave a comment here explaining why we're not expecting proveAttachment to work in this case?